### PR TITLE
[xs] add markdown format of the readme

### PR DESCRIPTION
**Pull request by subhankarb**
The readme section at top of data package view page
was not markdown enabled. This commit will enable the
readme section to be viewed as markdown.

Resolves - #270

### CODE REVIEW

File: app/templates/_snippets.html [hunk 1]
```diff
@@ -34,7 +34,7 @@
     <div class="col-md-8">
       <img src="{{dataset.image or 'https://assets.okfn.org/p/data/img/icon-128.png'}}"
         alt="{{dataset.title}}" class="logo" />
-      <div class="description">{{dataset.readme|truncate}} <a href="#readme">Read more</a></div>
+      <div class="description">{{dataset.readme|truncate|markdown}} <a href="#readme">Read more</a></div>
```
Comment (rufuspollock) [inline app/templates/_snippets.html]: Are you sure this is the right way round? You want this to be plain text with no markdown formatting i think -- and what happens if you truncate in the middle of markdown formatting ...

**Comment by rufuspollock** [comment]
@subhankarb do we have a small test for this?

**Comment by rufuspollock** [comment]
Where is de-htmlification as in the issue?

You don't want any markdown at the top of the page -- think about it, it will look really weird to have e.g.

> This is a XXX datasets
>
> ## A sudden head ...
>
> Read more

**Comment by subhankarb** [comment]
@rufuspollock Updated the de-htmlify code.

**Pull request merged**